static/css/styles.css

```diff
@@ -0,0 +1,3 @@
+body {
+    margin: 20px;
+}
```

Comment:
Add empty line. It is a nit, but we should always add empty line in the end of the file, so github won't show us these red circles

Reply:
Ok thank you for answer